scripts/run_validator.sh

```diff
@@ -1,5 +1,5 @@
 #!/usr/bin/env bash
 
-branch=latest
+branch=consistent-value-refs
```

Comment:
Reminder to update this branch once https://github.com/ONSdigital/eq-questionnaire-validator/pull/14 is merged